openslides_backend/presenter/get_user_scope.py

```diff
@@ -0,0 +1,55 @@
+from enum import Enum
+from typing import Any, Dict
+
+import fastjsonschema
+
+from ..shared.mixins.user_scope_mixin import UserScopeMixin
+from ..shared.schema import schema_version
+from .base import BasePresenter
+from .presenter import register_presenter
+
+get_user_scope_schema = fastjsonschema.compile(
+    {
+        "$schema": schema_version,
+        "type": "object",
+        "title": "get_user_related_models",
+        "description": "get user ids related models",
+        "properties": {
+            "user_ids": {
+                "type": "array",
+                "item": {"type": "integer"},
+            },
+        },
+        "required": ["user_ids"],
+        "additionalProperties": False,
+    }
+)
+
+
+class UserScope(int, Enum):
```

Comment:
You should import this from the shared folder

Reply:
Done.